packages/cli/bin/cli.js

```diff
@@ -12,64 +12,117 @@ const debug = require('../lib/debug')();
 const minimist = require('minimist');
 const path = require('path');
 const yeoman = require('yeoman-environment');
+const PREFIX = 'loopback4:';
 
-const opts = minimist(process.argv.slice(2), {
-  alias: {
-    version: 'v',
-    commands: 'l',
-  },
-});
-
-if (opts.version) {
-  const ver = require('../package.json').version;
-  console.log('Version: %s', ver);
-  return;
+/**
+ * Parse arguments and run corresponding command
+ * @param env Yeoman env
+ * @param {*} opts Command options
+ * @param log Log function
+ * @param dryRun flag for dryRun (for testing)
+ */
+function runCommand(env, opts, log, dryRun) {
+  const args = opts._;
+  const originalCommand = args.shift();
+  let command = PREFIX + (originalCommand || 'app');
+  const supportedCommands = env.getGeneratorsMeta();
+  if (!(command in supportedCommands)) {
+    command = PREFIX + 'app';
+    args.unshift(originalCommand);
+    args.unshift(command);
+  } else {
+    args.unshift(command);
+  }
+  debug('invoking generator', args);
+  // `yo` is adding flags converted to CamelCase
+  const options = camelCaseKeys(opts, {exclude: ['--', /^\w$/, 'argv']});
```

Comment:
Is it overkill to add tests for the excluded commands?

Reply:
@raymondfeng ^

Reply:
We might be able to get rid of that as we don't use `yo`.